SkyEditor.RomEditor.Rtdx/Domain/Rtdx/Constants/DungeonIndex.cs

```diff
@@ -0,0 +1,152 @@
+﻿using SkyEditor.RomEditor.Infrastructure.Automation.CSharp;
+using SkyEditor.RomEditor.Infrastructure.Automation.Lua;
+using SkyEditor.RomEditor.Infrastructure;
+using System;
+using System.Collections.Generic;
+
+namespace SkyEditor.RomEditor.Domain.Rtdx.Constants
+{
+    public class DungeonIndexLuaExpressionGenerator : ILuaExpressionGenerator
```

Comment:
My code isn't the best example of consistency, but it'd be best if either both code generators were in this file or if they were both in the same file as the generators, and not a mix of the two.

But I'm definitely guilty of this too.